fibers.cpp

```diff
@@ -46,10 +46,9 @@
 #define LOG(stuff)
 
 static int fiber_count = 0;
-

```

Comment:
not intentionally, I run into a git conflict problem before, probably accidentally removed the line

I will put it back.